src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -406,6 +406,31 @@ public MethodInfo(AssemblyInfo assembly, MethodDefinitionHandle methodDefHandle,
             localScopes = pdbMetadataReader.GetLocalScopes(methodDefHandle);
         }
 
+        public List<ParameterInfo> GetMethodParams()
```

Comment:
This is already in `MethodInfo` class.
```suggestion
        public List<ParameterInfo> GetParameters()
```